ConwaysGameOfLife.Tests/OutputFormatterShould.cs

```diff
@@ -5,78 +5,38 @@ namespace ConwaysGameOfLife.Tests
 {
     public class OutputFormatterShould
     {
+        OutputFormatter formatter;
+        public OutputFormatterShould()
+        {
+            formatter = new OutputFormatter();
+        }

```

Comment:
private field naming convention

```suggestion
        private OutputFormatter _formatter;
        
        public OutputFormatterShould()
        {
            _formatter = new OutputFormatter();
        }
```